src/Events/QR/Routes.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * The Routes class for the QR module.
+ *
+ * @since TBD
+ */
+
+namespace TEC\Events\QR;
+
+use TEC\Common\Contracts\Provider\Controller;
+use Tribe__Events__Rewrite;
+
+/**
+ * Class Routes.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\QR
+ */
+class Routes extends Controller {
+	/**
+	 * The route prefix for QR codes.
+	 *
+	 * @since TBD
+	 * @var string
+	 */
+	private $route_prefix = 'qr';
```

Comment:
We set it here buts we overwrite it during registration ? Maybe not set it at all ?